packages/iterator-utilities/src/lib/takeWhile.ts

```diff
@@ -0,0 +1 @@
+export { filter as takeWhile } from './filter';
```

Comment:
Can't you just export this frm `filter` similar to how you exported other aliases?